magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -0,0 +1,41 @@
+"""
+Module for generating bash script lines for running analysis in different clusters
+"""
+
```

Comment:
I think here you should also add:

```python
__all__ =  ["slurm_lines"]
```

Reply:
ok, I will add it, but note that it does not change anything, because by default  all functions are exported